### ISSUE TITLE
Access to inverses when clicking on a cell

### ISSUE THREAD

**Issue by turion** [jamievicary/globular#59]
A few times I wanted to attach the inverse of an invertible cell, by clicking on that cell. But this doesn't work. The only way to attach an inverse seems to be to click into the current cell and selecting the inverse. It would be great if the former way of attaching an inverse would work as well somehow.

**Comment by jamievicary** [comment]
Hi, I agree this would be nice. The problem is that if you click on a cell
in the signature and get a big list of possibilities, how do I know which
are the cell and which are the inverse? Some extra information would be
needed and I'm not sure quite how to represent it.

**Comment by turion** [comment]
Yes, that's not obvious at all. Especially for endocells, Globular won't be able to work out which way to attach it. Two ideas for that:

- Display places where the inverse cell can be attached in a different colour than yellow (e.g. light green or blue).
- When clicking on a cell in the left menu, the inverse cell appears directly below and indented (so the cell becomes a tree with one leaf which is the inverse).

**Comment by jamievicary** [comment]
Different colours could also be confusing --- how is a new user supposed to
know which is which?

I think that when you click, you could just get two lists underneath, not
one: a list of places to attach the cell, and a list of places to attach
the inverse.

**Comment by turion** [comment]
Right, for a new user the colours may be confusing. Your second option sounds very good.
